avatar.css

```diff
@@ -0,0 +1,41 @@
+:root {
```

Comment:
Any specific reason for declaring variables ? Because, Variables haven't used.

Reply:
no the root i used just to check it's functioning with badges root and few things will be moved to utils or main css.